src/components/ui/Menus/ContextMenu.tsx

```diff
@@ -36,14 +42,26 @@ export default function ContextMenu({
   y,
   menuRef,
   selectedFiles,
-  handleFavoriteToggleMenuItemClick,
+  handleContextMenuFavorite,
   setShowPropertiesDrawer,
   setShowContextMenu,
   setShowRenameDialog,
   setShowDeleteDialog,
   setShowPermissionsDialog,
   setShowConvertFileDialog
 }: ContextMenuProps): React.ReactNode {
+  const { fileBrowserState } = useFileBrowserContext();
+  const { folderPreferenceMap } = usePreferencesContext();
+
+  const isFavorite: boolean = folderPreferenceMap[
+    makeMapKey(
+      'folder',
+      `${fileBrowserState.currentFileSharePath?.name}_${selectedFiles[0].path}`
+    )
+  ]
+    ? true
+    : false;
```

Comment:
A more idiomatic way of doing this conversion would be to either use the `!!` (double bang) operator, or the `Boolean()` constructor.